internals/fetchmetrics.py

```diff
@@ -250,10 +250,8 @@ def get_template_data(self):
 
     # Save bucket ids for each histogram type, FeatureObserver and
     # MappedCSSProperties.
-    for histogram_id in self.MODEL_CLASS.keys():
-      enum = list(filter(
-          lambda enum: enum.attributes['name'].value == histogram_id,
-          enum_tags))[0]
+    for histogram_id in list(self.MODEL_CLASS.keys()):
+      enum = list([enum for enum in enum_tags if enum.attributes['name'].value == histogram_id])[0]
```

Comment:
I think the change here is `filter`

Reply:
Yeah, one way to deal with `filter(...)` is to make it `list(filter(...))`, and the other way is to change it to a list comprehension `[x for x in y if cond]`.  But, I don't think that we need to do both because a list comprehension always results in a list.

Reply:
```suggestion
      enum = [enum for enum in enum_tags if enum.attributes['name'].value == histogram_id][0]
```